app/components/Home.vue

```diff
@@ -76,13 +77,13 @@ export default {
   },
   created () {
     // cuando se vuelve del background o arranca la aplicacion siempre se muestra el primer tab
```

Comment:
Since **created** is a vue lifecycle i guess it's known what the effect is. I'd remove this comment.